docker/init.sls

```diff
@@ -85,8 +85,8 @@ docker package repository:
 {%- endif %}
 
 docker package:
-  {%- if "version" in docker %}
   pkg.installed:
+  {%- if "version" in docker %}
     {%- if grains["oscodename"]|lower == 'jessie' %}
     - name: docker.io
     - version: {{ docker.version }}
```

Comment:
isn't this the same as  docker.pkg.version ?

Reply:
No, that `docker` is the construct created by map.jinja which calculates `version` from the `docker-pkg:version` pillar. But `docker-pkg` namespace is, or should be, depreciated.

There is some repetition in the code block - I could fix that but small PR gets too big then?